Vineyard/Vineyard/ProfileView.swift

```diff
@@ -22,37 +22,27 @@ struct ProfileView: View {
                     }) {
                         Text("Sign out")
                     }
-                    HStack {
+                    HStack(spacing: 20) {
                         Image(systemName: "person.crop.circle.fill")
                             .resizable()
-                            .frame(width: 75, height: 75)
+                            .frame(width: 70, height: 70)
                             .foregroundColor(.gray)
                         VStack(alignment: .leading, spacing: 2) {
                             Text(loginViewModel.currentUser?.name ?? "no name")
                                 .font(.system(size: 20, weight: .bold))
                             HStack() {
-                                Text("x Friends")
-                                    .foregroundColor(.gray)
-                                Image(systemName: "circle.fill")
-                                    .resizable()
-                                    .frame(width: 3, height: 3)
-                                    .foregroundColor(.gray)
-                                if(viewModel.user.groups.count > 1) {
-                                    Text("\(viewModel.user.groups.count) groups")
-                                        .font(.system(size: 16))
-                                        .fontWeight(.regular)
-                                        .foregroundColor(.gray)
-                                } else {
-                                    Text("\(viewModel.user.groups.count) group")
-                                        .font(.system(size: 16))
-                                        .fontWeight(.regular)
-                                        .foregroundColor(.gray)
-                                }
+                                let count = viewModel.user.groups.count
+                                Text("\(count) \(count > 1 ? "groups" : "group")")
+                                     .font(.system(size: 16))
+                                     .fontWeight(.regular)
+                                     .foregroundColor(.gray)
                             }
                         }
                     }
                     .frame(maxWidth: .infinity, alignment: .leading)
-                    .padding(.top, 44)
+                    .padding()
+                    .background(Color(red: 0.937, green: 0.937, blue: 0.937))
```

Comment:
We should store hardcoded colors in an extensions file. You should create a group called "Extensions", and in that a file called "Color+Extensions":

```swift
// in Color+Extensions.swift

extension Color {
    static var profileViewCellBackground: Color {
        Color(red: 0.937, green: 0.937, blue: 0.937)
    }
}
```
and then:

```suggestion
                    .background(Color.profileViewCellBackground)
```

Reply:
Also, is this the same color used in dark mode?

Reply:
i'm not too sure if the updated push accounts for light/dark mode, can you lmk if it does not?

Reply:
It won't change dynamically, but that's fine for now.